src/Components/Publishing/Display/Canvas/CanvasVideo.tsx

```diff
@@ -18,12 +18,14 @@ export class CanvasVideo extends React.Component<VideoProps, any> {
     this.state = { isPlaying: false }
   }
 
```

Comment:
I wonder if it makes sense to use once or some other kind of filtering of event here -- looks like this will fire on both play and pause.